.coveragerc

```diff
@@ -0,0 +1,9 @@
+[coverage:report]
+omit =
+    */test_*
+    relation_engine_server/api_versions/*
+    relation_engine_server/exceptions.py
+    relation_engine_server/main.py
+    relation_engine_server/utils/auth.py
+    relation_engine_server/utils/bulk_import.py
+    relation_engine_server/utils/parse_json.py
```

Comment:
Why are these omitted?

Reply:
Because those files are not included in the original coverage, if our goal is not to produce the original coverage, we don't need to omit them.

Reply:
How are they excluded from the original coverage?

Reply:
The original coverage was produced when I added `coverage report --omit=*/test_*` in https://github.com/kbase/relation_engine/pull/184/commits/64926da1f1562ce4bc5fe12d56cd42ea4b684f08. I am not sure why they were excluded from the original coverage, even though most of these files have 0% coverage.

Reply:
So there's nothing in the configuration that excludes them in the prior setup? Just `coverage` is dropping them for some reason?

Reply:
I did not find any coverage configuration in the codebase that excluded them in the prior setup. What are your thoughts on this? Should we include them if there are no particular reasons to omit these files?

Reply:
We definitely shouldn't seemingly arbitrarily exclude files just because the prior setup excluded them

Reply:
👍